procurement_auto_create_group/models/procurement_group.py

```diff
@@ -13,13 +13,6 @@ def _get_rule(self, product_id, location_id, values):
         rule = super()._get_rule(product_id, location_id, values)
         # If there isn't a date planned in the values it means that this
         # method has been called outside of a procurement process.
-        if (
-            rule
-            and not values.get("group_id")
```

Comment:
Are you sure we want to remove this check? This could have side-effects on existing installations no?

Reply:
Following your commit message it's what you want, but just to be sure about the possible regressions.

Reply:
I could not find out why it was there. I hope it is safe to remove it @LoisRForgeFlow @JordiBForgeFlow 
Otherwise it prevents the feature to work in a pull flow coming from a SO.

Reply:
Just in case, I think you can reduce the risk by putting the if condition in a method and doing super in the new module, removing the condition that is not wanted.

Reply:
If you don't see a breaking change, let's consider it a bug fix.
Note that I'm not changing anything in regards of the above comment regarding date planned.

The configuration of the rule should prevail. If it is configured to create a new procurement group, then it should do it. Independently of how the rule is run.